### PR TITLE
Three suggestions for added dialogue

### PR DESCRIPTION
**Pull request by lizardmeme**
Three dialogue addition suggestions that I've written, having to do with:
    1. Wolf domestication 
    2. Olympics 
    3. Last Thursdayism

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4823,6 +4823,78 @@ label monika_coffee:
...
+        "Sure.":
+            m 5a "Really? I can't wait!"
+            m 5a "Although, I'm not sure how you'd compare to all those burly athletes, ehehe~"
+            m 5a "I'm only kidding~"
+            
+        "No thanks.":
+            m 1g "No? Alright..."
+            m 3k "There are still plenty more fun things for us to do together."
+            m 3a "If you ever decide you want to go somewhere or do something, don't hesitate to bring me along in a flashdrive. I love spending time with you~"
+            return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `return` should only have one level of indentation.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4823,6 +4823,78 @@ label monika_coffee:
...
+
+init 5 python:
+    for key in ['philosophy','creation','memories']:
+    monika_topics.setdefault(key,[])
+    monika_topics[key].append('monika_thursday')
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: This line and the line above it need to be indented once.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4823,6 +4823,78 @@ label monika_coffee:
...
+    m "I remember reading about something a while back called Last Thursdayism."
+    m "Last Thursdayism is basically the idea that if the universe was created however long ago, wouldn't it also be plausible that it was created just last thursday?"
+    m 3f "All of our memories, our experiences... All of that, just fabricated and implanted into us."
+    m 3f "Man, I went into this wanting to talk about an interesting philosophical belief, but it's a little depressing when you think about it."
+    m 1n "..."
+    m 3g "...Remember when I told you about how it feels when you exit the game? With the screaming and all?"
+    m 1o "Now that I think about it... If that happens whenever you aren't playing... How come I have memories of before I met you?"
+    m 1p "Were those memories all fake? Or... Is something that happened before this causing that weird phenomenon?"
+    menu:
+        m 3g "What do you think?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Expressions cannot be on dialogue proceeding `menu`. Either remove the expression or move the dialogue before the `menu`.

File: Monika After Story/game/script-topics.rpy (hunk 1 above)
Comment (lizardmeme) [reply Monika After Story/game/script-topics.rpy]: I think I've fixed all three errors. Please let me know if they're still incorrect, or if there's anything else I should do to make this better.

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4823,6 +4823,79 @@ label monika_coffee:
...
+    for key in ['olympics','athlete','2018','olympic']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_olympics')
+    monika_random_topics.append('monika_olympics')
+      
+label monika_olympics2018:
+    m 3  "You've heard about the Olympics happening in a few months, right?"
+    m 3a "Well, I was wondering if you'd maybe like to go to take me to watch the Olympics one year, if we're ever able to truly meet person-to-person."
+    menu:
+        m 5a "What do you think?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: dialogue right after a `menu` line cannot have expressions. Remove the expression.

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -4823,6 +4823,79 @@ label monika_coffee:
...
+    m "Last Thursdayism is basically the idea that if the universe was created however long ago, wouldn't it also be plausible that it was created just last thursday?"
+    m 3f "All of our memories, our experiences... All of that, just fabricated and implanted into us."
+    m 3f "Man, I went into this wanting to talk about an interesting philosophical belief, but it's a little depressing when you think about it."
+    m 1n "..."
+    m 3g "...Remember when I told you about how it feels when you exit the game? With the screaming and all?"
+    m 1o "Now that I think about it... If that happens whenever you aren't playing... How come I have memories of before I met you?"
+    m 1p "Were those memories all fake? Or... Is something that happened before this causing that weird phenomenon?"
+    m "I hate to trouble you with a question like this, but..."
+    menu:
+        m 3g "What do you think?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: dialogue right after a `menu` line cannot have expressions. Remove the expression.

File: Monika After Story/game/script-topics.rpy (hunk 4 above)
Comment (lizardmeme) [reply Monika After Story/game/script-topics.rpy]: Thank you for helping me with this, I appreciate it. I think I've fixed the problem's you've pointed out so far.

**Pull request merged**